examples/test_data_management.py

```diff
@@ -3,6 +3,10 @@
 #
 # 1. Locustfile parse time
 # 2. Locust start (init)
+# M1. CPU & memory usage
```

Comment:
What's "M"? :) Its kind of arbitrary where you put these, but I'd prefer having them after 5.

Reply:
`M` as in "monitor events", i was abit unsure where to place them in the list and what to name them, it was `X` first, but then I saw that there already was one "numbered" with that.

I put them after 2 since they will start to appear after 2, I can for sure move them after 5, where I actually put them before I ran the test :)